src/payment/payment-request-transformer.ts

```diff
@@ -45,8 +45,8 @@ export default class PaymentRequestTransformer {
         }
 
         return {
+            additionalAction: this._getAdditionalAction(payment.paymentRecaptchaToken),
```

Comment:
The same `additionalAction` object needs to be added to the return value of `transformWithHostedFormData` otherwise the value won't flow through to the hosted form.